modules/local/process/multiqc_custom_fail_mapped.nf

```diff
@@ -0,0 +1,35 @@
+// Import generic module functions
+include { saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+
+process MULTIQC_CUSTOM_FAIL_MAPPED {
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+
+    conda (params.enable_conda ? "conda-forge::sed=4.7" : null)
+    if (workflow.containerEngine == 'singularity' && !params.pull_docker_container) {
+        container "https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img"
```

Comment:
This is the base container from biocontainers right? 

Not sure why we need a separate multiqc_custom_fail_mapped process and can't have this in one multiqc one?

Reply:
Ah, mainly in the name of modularity! So, all of the different MultiQC checks will be run in different parts of the workflow so its better to keep them separate. Also makes it easier to share the modules. I used that Biocontainer for any process in the pipeline that only requires a vanilla Linux installation as is the case here. For one of the other MultiQC specific custom processes I need [`Python`](https://github.com/nf-core/rnaseq/blob/ff1aa1c60b3e4156bc4c0bcd50fbd7b0407b8663/modules/local/process/multiqc_custom_biotype.nf#L15).

Reply:
PS: I was very tempted to just dump things in the same process but then I couldn't sleep at night anymore....